lib/webdriveragent.js

```diff
@@ -25,6 +24,37 @@ const WDA_AGENT_PORT = 8100;
 const WDA_BASE_URL = 'http://localhost';
 const BUILD_TEST_DELAY = 1000;
 
+
+class NoSessionProxy extends JWProxy {
```

Comment:
Ah, yes, you'll need the util module from `appium-support`. 

If it is in the same directory you should be able to import with either of the two:
```js
import NoSessionProxy from './no-session-proxy';
import { NoSessionProxy } from './no-session-proxy';
``